api/hub/src/datasources/moderation-decision.ts

```diff
@@ -177,7 +205,9 @@ class ModerationDecisionAPI extends DataSource {
     if (decision.moderator) {
       const profileAPI = new ProfileAPI({ dbID: this.dbID, collection: 'Profiles' });
       const moderator = await profileAPI.getProfile(decision.moderator);
-      decision = Object.assign({}, decision, moderator);
+      decision = Object.assign({}, decision, { 
+        "moderatorProfile": moderator
```

Comment:
the attribute here doesn't need to be a string, it can be `{moderatorProfile: moderator}`